appveyor.yaml

```diff
@@ -0,0 +1,22 @@
+version: '{build}'
+branches:
+  only:
+  - master
+configuration: RelWithDebInfo
+build_script:
+- ps: >-
+    choco install cmake
+    choco install python2
+    mkdir build
+    cd build
+    cmake -D ENABLE_TRACE=ON -D BUILD_CLAR=ON -D STDCALL=ON .. -G"Visual Studio 11"
+    cmake --build . --config RelWithDebInfo
+test_script:
+- ps: ctest -V .
+notifications:
+- provider: Email
+  to:
+  - emeric.fermas@gmail.com
```

Comment:
I'm ok with tthat. However, I'm not sure I'll be available enough to paste them in real time in the various usual channels :stuck_out_tongue_winking_eye: